Source/WebCore/platform/network/ResourceRequestBase.h

```diff
@@ -30,6 +30,7 @@
 #include "FormData.h"
 #include "FrameLoaderTypes.h"
 #include "HTTPHeaderMap.h"
+#include "IPAddressSpace.h"
```

Comment:
It would be nice to forward declare and put implementations into cpp files instead of including a header from this very commonly included header and increasing the build time.